stl/inc/filesystem

```diff
@@ -3750,6 +3750,10 @@ namespace filesystem {
 
     // FUNCTION create_directories
     inline bool create_directories(const path& _Path, error_code& _Ec) {
+        if (_Path.empty()) {
+            _Throw_fs_error("create_directories", _Make_ec(__std_win_error::_Invalid_parameter), _Path);

```

Comment:
Filesystem functions that have an `error_code` argument should usually use that to indicate errors, rather than throwing. See [N4861](https://wg21.link/n4861) [fs.err.report].

Reply:
Thanks. That's right. also I chose `__std_win_error::_Path_not_found` as the thrown error and also as the `error_code` just to make it consistent with the `create_directory` behavior on the same input. Please let me know if it's not the right choice.

Reply:
Looks good to me.